### DISCUSSION TITLE
Pi pico w

### DISCUSSION THREAD

**Discussion by PlasticBurners** [wiredopposite/OGX-Mini#48] [General]
Could the Bluetooth on a pico pi w be used for Xbox one and PS5 controllers

**Comment by Invictaz** [comment]
Bluetooth is in development by @wiredopposite iirc

**Comment by Invictaz** [comment]
Bluetooth is now there in v1.0 alpha

**Reply by PlasticBurners** [reply]
Yes mate I set it up yesterday works great 👍

**Comment by EatPrilosec** [comment]
> Bluetooth is now there in v1.0 alpha

Is the picoW version ONLY Bluetooth? I cant get a USB controller to work on the picoW

**Reply by Invictaz** [reply]
You need to flash the non W firmware for now. Usb and Bluetooth simultaneously does not work for now.

**Reply by EatPrilosec** [reply]
flashing the pico firmware also seems not to work either (alpha3)

**Reply by EatPrilosec** [reply]
> You need to flash the non W firmware for now. Usb and Bluetooth simultaneously does not work for now.

when i flash pico firmware on the picow, the light doesnt even come on
